src/components/Integrations/Shopify/Shopify.test.js

```diff
@@ -152,4 +169,98 @@ describe('Shopify Component', () => {
+    // Arrange
+    const experimentalFeaturesData = {
+      DopplerAPI: { apikey: 'myapikey', listId: 455222 },
+    };
+    const storage = new FakeLocalStorage();
+    storage.setItem('dopplerExperimental', JSON.stringify(experimentalFeaturesData));
+    const experimentalFeatures = new ExperimentalFeatures(storage);
+    const listExist = {
+      success: true,
+      value: {
+        name: 'Shopify Contacto',
+        id: 27311899,
+        amountSubscribers: 200,
+        state: SubscriberListState.ready,
+      },
+    };
+
+    const shopifyClientDouble = {
+      getShopifyData: async () => oneShopConnected,
+    };
+    const dopplerAPIClientDouble = {
+      getListData: async () => listExist,
+    };
+
+    // Act
+    const { container, getByText } = render(
+      <AppServicesProvider
+        forcedServices={{
+          shopifyClient: shopifyClientDouble,
+          dopplerApiClient: dopplerAPIClientDouble,
+          experimentalFeatures: experimentalFeatures,
+        }}
+      >
+        <DopplerIntlProvider>
+          <Shopify />
+        </DopplerIntlProvider>
+      </AppServicesProvider>,
+    );
+
+    // Assert
+    expect(container.querySelector('.loading-box')).toBeInTheDocument();
+    await waitForDomChange();
+    expect(container.querySelector('.dp-integration__status')).toBeInTheDocument();
+    expect(getByText(listExist.value.amountSubscribers.toString()));
+  });
+
+  it('should work ok whith api client with token when feature is enabled without apikey', async () => {
+    // Arrange
+    const experimentalFeaturesData = {
+      DopplerAPI: { listId: 455222 },
+    };
+    const storage = new FakeLocalStorage();
+    storage.setItem('dopplerExperimental', JSON.stringify(experimentalFeaturesData));
+    const experimentalFeatures = new ExperimentalFeatures(storage);
+    const listExist = {
+      success: true,
+      value: {
+        name: 'Shopify Contacto',
+        id: 27311899,
+        amountSubscribers: 200,
+        state: SubscriberListState.ready,
+      },
+    };
+
+    const shopifyClientDouble = {
+      getShopifyData: async () => oneShopConnected,
+    };
+    const dopplerAPIClientDouble = {
+      getListData: async () => listExist,
+    };
+
+    // Act
+    const { container, getByText } = render(
+      <AppServicesProvider
+        forcedServices={{
+          shopifyClient: shopifyClientDouble,
+          dopplerApiClient: dopplerAPIClientDouble,
+          experimentalFeatures: experimentalFeatures,
+        }}
+      >
+        <DopplerIntlProvider>
+          <Shopify />
+        </DopplerIntlProvider>
+      </AppServicesProvider>,
+    );
+
+    // Assert
+    expect(container.querySelector('.loading-box')).toBeInTheDocument();
+    await waitForDomChange();
+    expect(container.querySelector('.dp-integration__status')).toBeInTheDocument();
+    expect(getByText(listExist.value.amountSubscribers.toString()));
```

Comment:
Here you are simply searching for `200`. I think that having false positives is too easy.

Reply:
What do you suggest?

Reply:
Mmm, maybe find the element and check the value? or, does it have related content? in that case, you could search for something like `200 subscribers`

Reply:
I don't have something like that O_o this is a table cell, with nothing subscribers amount as a number, like this:  `<td>200</td>`

Reply:
In that case, do not worry, we could deal with this issue later.